### PR TITLE
Kept exception messages consistent, recalculated Vx before updating w

### PR DESCRIPTION
**Pull request by serickso**

### CODE REVIEW

**Comment by haifengl** [comment]
why do we need updatedVx? It is same as Vx, right?

**Comment by serickso** [comment]
It changes because of the update to V. First we update V, then use the updated V to update w

File: core/src/main/java/smile/regression/RLS.java [hunk 1]
```diff
@@ -209,12 +214,17 @@ public void learn(double[] x, double y) {
             for (int i = 0; i <= p; i++) {
                 double tmp = V.get(i, j) - ((Vx[i] * Vx[j])/v);
                 V.set(i, j, tmp/lambda);
+                
+                if (j == 0){
+                    updatedVx[i] = 0;
+                }
+                updatedVx[i] += V.get(i, j) * x1[j];
             }
         }
```
Comment (haifengl) [inline core/src/main/java/smile/regression/RLS.java]: just add 
```
V.ax(x1, Vx)
```
here. And remove updatedVx.

**Comment by haifengl** [comment]
don't do matrix operation by yourself. ax() is much more efficient.

**Comment by haifengl** [comment]
Thank you!

**Pull request merged**